src/commands/new-proj.ts

```diff
@@ -150,6 +225,34 @@ async function initDefaultProject() {
   }
 }
 
+/**
+ * Attachs possibly wanted components to Minikube
+ * 
+ * @param compsArr Array of components
+ * @param image Image for component, if any
+ * @param port Port for component (Pod)
+ * @param portType Port type for port (Service)
+ * @param ports Ports for component (Service/Deployment)
+ * @param replicas Replicas of component, if any
+ */
+async function attachToMinikube(compsArr: string[], image: string, port: number, portType: string, ports: Array<unknown>, replicas: number) {
+  const componentsArr: KubeComponent[] = [];
+  for (const comp in compsArr) {
+    componentsArr.push({
+      args: {
+        name: `${projName}-${comp}`,
+        image: image,
+        port: port,
+        portType: portType,
+        ports: ports,
+        replicas: replicas
+      },
+      type: comp
+    });
+  }
+  MinikubeUtils.createComponents(componentsArr, repoPath);
+}
+
 /**
  * Runs command new-repo with user given parameters
  */
```

Comment:
This file could maybe benefit from breaking up too.

Reply:
Creating project utils is a matter of another issue.